orchagent/portsorch.cpp

```diff
@@ -4175,6 +4180,38 @@ void PortsOrch::generatePriorityGroupMapPerPort(const Port& port)
     CounterCheckOrch::getInstance().addPort(port);
 }
 
+void PortsOrch::generatePortCounterMap()
+{
+    if (m_isPortCounterMapGenerated)
+    {
+        return;
```

Comment:
should you add a log in NOTICE for debug purposes? are we expecting this to happen a lot?

Reply:
on the "setCounterIdList" function which eventually will be called there is a log for debugging already:
SWSS_LOG_DEBUG("Added flex stat '%s' to object '%s'", counter_stat.c_str(), sai_serialize_object_id(object_id).c_str());

I thought it will be redundant to add another logging here.